pyproject.toml

```diff
@@ -46,7 +46,7 @@ all = [
     "aiosqlite",
     "alembic",
     "anyio",
-    "appdirs",
+    "platformdirs",
```

Comment:
The deps are sorted alphabetically in each section. Would you move these accordingly?